src/api/uniqueSdk/types.ts

```diff
@@ -59,8 +59,11 @@ export interface NFTToken {
   description?: string
   collectionCover?: string
   isAllowed?: boolean
+  video: VideoAttribute
 }
 
+export type VideoAttribute = ({ url: string; urlInfix?: undefined; ipfsCid?: undefined } & { hash?: string } & { fullUrl: string | null }) | ({ urlInfix: string; url?: undefined; ipfsCid?: undefined } & { hash?: string } & { fullUrl: string | null }) | ({ ipfsCid: string; url?: undefined; urlInfix?: undefined } & { hash?: string } & { fullUrl: string | null }) | undefined;
```

Comment:
don't we hae this type in SDK?

type declaration should always be before usage.

Reply:
fixed